tests/handlers/test_combined_data.py

```diff
@@ -7,58 +7,104 @@
 
 
 def test_load(va_governor_county_data):
+    election_id = "2017-11-07_VA_G"
+    office_id = "G"
+    geographic_unit_type = "county"
     estimands = ["turnout"]
+    estimand_baselines = {"turnout": "turnout"}
     live_data_handler = MockLiveDataHandler(
-        "2017-11-07_VA_G", "G", "county", estimands=["turnout"], data=va_governor_county_data
+        election_id, office_id, geographic_unit_type, estimands=["turnout"], data=va_governor_county_data
     )
     current_data = live_data_handler.data
+    preprocessed_data_handler = PreprocessedDataHandler(
+        election_id,
+        office_id,
+        geographic_unit_type,
+        estimand_baselines=estimand_baselines,
+        estimands=estimands,
+        data=va_governor_county_data,
+    )
+    preprocessed_data = preprocessed_data_handler.data
 
     combined_data_handler = CombinedDataHandler(
-        va_governor_county_data, current_data, estimands, "county", handle_unreporting="drop"
```

Comment:
Is there a reason why we can't reuse or update `va_governor_county_data` to be the required `PreprocessedData` here (and in other similar unit tests)?  I think this test is testing too many things, as in, `PreprocessedDataHandler` and `CombinedDataHandler` when it should only be testing `CombinedDataHandler`.  Or we could create another test 🤔

Reply:
good call! I updated the tests so now we modify `va_governor_county_data` so that it has the columns that `CombinedDataHandler` expects

Reply:
Nice!  Thanks! 🎉 🎉